frontend/src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,100 @@
+
+import React, {useState} from 'react';
+import { View, Text, TouchableOpacity, Image, StyleSheet} from 'react-native';
+import FormInput from '../components/FormInput'
+import FormButton from '../components/FormButton';
+import SocialButton from '../components/SocialButton';
+
+
+const LoginScreen = ({navigation}) => {
+  const [email, setEmail] = useState();
+  const [password, setPassword] = useState();
+    return (
+        <View style={styles.container}>
+            <Image 
+              style = {styles.logo} 
+              source={require('../../assets/friends.png')}
+            />
+            <FormInput
+              labelValue={email}
+              onChangeText={(userEmail) => setEmail(userEmail)}
+              placeholderText="Email"
+              iconType="user"
+              keyboardType="email-address"
+              autoCapitalize="none"
+              autoCorrect={false}
+            />
+            <FormInput
+              labelValue={password}
+              onChangeText={(userPassword) => setPassword(userPassword)}
+              placeholderText="Password"
+              iconType="lock"
+              secureTextEntry={true}
+            />
+            <FormButton
+              buttonTitle="SIGN IN"
+              onPress={() => navigation.navigate('Home')}
+            />
+
+            <TouchableOpacity style={styles.forgotButton} onPress={() => {}}>
+              <Text style={styles.navButtonText}>FORGOT PASSWORD?</Text>
+            </TouchableOpacity>
+
+            <Text style={styles.textRegister}> ────── OR  ──────</Text>
+
+            <SocialButton 
+              buttonTitle="SIGN IN USING GOOGLE"
+              btnType="google"
+              color="#de4d41"
+              backgroundColor="#f5e7ea"
+              onPress={() => {}}
```

Comment:
Your social button doesn't take an onPress -> this goes in the ...rest props -> would it be clearer is this was a named prop - otherwise we might not be able to find the correct prop name